### DISCUSSION TITLE
[Feature] Sort options for people list

### DISCUSSION THREAD

**Discussion by jhf2442** [immich-app/immich#6053] [Feature Request]
### The feature

Request to have sort options for the people list. Currently this seems to be according to the number of tagged pictures ?
Would be good to be able to sort alphabetically

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by kingvuke** [comment]
see https://github.com/immich-app/immich/discussions/5927

**Reply by jhf2442** [reply]
you're right, didn't find it when looking for similar tickets. Closing this one
